app/jobs/cancel_reservations_job.rb

```diff
@@ -0,0 +1,16 @@
+class CancelReservationsJob < ApplicationJob
+  queue_as :default
+
+  def perform(*args)
+    #find all screenings that will start in 30 minuts or earlier
+    screenigs  = Screenings::UseCases::FindAll.new.call
```

Comment:
I think in this job screen is not important, you could search for all reservation without looking for the screen

Reply:
But I need  screening date to find out which reservations to cancel? And I am finding reservations based on screening id. Should I take it outside the job?

Reply:
ok, right. you want to run it half-hour before screening